docs/section4/4-3.md

```diff
@@ -0,0 +1,41 @@
+# Airside System – Rule 4-3
```

Comment:
I see bunch of text above this line - should it be removed?

Reply:
i believe that text should be removed.